src/GamesList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from "react";
+import fetchGames from "./apis/api";
+
+function GamesList() {
+  const [gamesList, setGamesList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
```

Comment:
Good to see you've got an isLoading state for a nicer user experience while waiting for the page to load 🤸